test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -euo pipefail
+
+VERSION=$1
+
+if [ -z "$VERSION" ];
+then
+    echo 'version number required'
+    exit 1
+fi
+
+function releaseToRegion {
```

Comment:
No idea what you are doing.
Are you actually deploying to s3 or is this in some SAM sandbox?